cmd/content-sources/main.go

```diff
@@ -1,25 +1,53 @@
 package main
 
 import (
-	"log"
+	"os"
 
 	"github.com/content-services/content-sources-backend/pkg/db"
 	"github.com/content-services/content-sources-backend/pkg/handler"
 	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	golog "github.com/labstack/gommon/log"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/ziflex/lecho/v3"
 )
 
 func main() {
+	e := echo.New()
+
+	logLevel := golog.ERROR
+
+	if false { //TODO: Dev debug based on env
+		logLevel = golog.DEBUG
+	}
```

Comment:
Lets go ahead and turn on console logging in this pr, and then make it configurable via viper integration.  I think just this:

log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})

and then passing  log.Logger instead of os.Stdout below in the lecho.New() call does it

Reply:
Nice.

Reply:
Addressed.